src/components/views/detail/components/body-editor/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import { Editor, EditorState } from "draft-js";
+
+// https://draftjs.org/docs/getting-started
```

Comment:
Could you explain what this package gives us? What is it's impact on over all size?

Reply:
nvm found it, https://bundlephobia.com/result?p=draft-js@0.10.5.

This is a quite a large package for a PWA. I see it's value but is it a cost we want? Open to your thoughts.  I understand we are not doing too much optimization before we are done but building around this may limit us in the future.